lib/initRedux.js

```diff
@@ -0,0 +1,32 @@
+import { createStore, applyMiddleware, combineReducers, compose } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import activeEvent from '../src/reducers/reducer_active_event';
+
+let reduxStore = null;
+
+function create(apollo, initialState = {}) {
+  return createStore(
+    combineReducers({
+      activeEvent: activeEvent,
+      apollo: apollo.reducer(),
+    }),
+    initialState,
+    compose(
+      applyMiddleware(apollo.middleware()),
+      composeWithDevTools(applyMiddleware(thunkMiddleware)),
```

Comment:
How would you make redux dev tools added only in development build?

Reply:
set environment variables and check if DEV before additing